### PR TITLE
feat: soundboard

### PR DESCRIPTION
**Pull request by sdanialraza**
**Please describe the changes this PR makes and why it should be merged:**
Supersedes #780

**If applicable, please reference Discord API Docs PRs or commits that influenced this PR:**
- https://github.com/discord/discord-api-docs/pull/6260
- https://github.com/discord/discord-api-docs/pull/7207

### CODE REVIEW

File: deno/gateway/v10.ts [hunk 1]
```diff
@@ -231,6 +244,15 @@ export enum GatewayDispatchEvents {
...
 	GuildRoleDelete = 'GUILD_ROLE_DELETE',
 	GuildRoleUpdate = 'GUILD_ROLE_UPDATE',
+	GuildScheduledEventCreate = 'GUILD_SCHEDULED_EVENT_CREATE',
+	GuildScheduledEventDelete = 'GUILD_SCHEDULED_EVENT_DELETE',
+	GuildScheduledEventUpdate = 'GUILD_SCHEDULED_EVENT_UPDATE',
+	GuildScheduledEventUserAdd = 'GUILD_SCHEDULED_EVENT_USER_ADD',
+	GuildScheduledEventUserRemove = 'GUILD_SCHEDULED_EVENT_USER_REMOVE',
+	GuildSoundboardSoundCreate = 'GUILD_SOUNDBOARD_SOUND_CREATE',
+	GuildSoundboardSoundDelete = 'GUILD_SOUNDBOARD_SOUND_DELETE',
+	GuildSoundboardSoundsUpdate = 'GUILD_SOUNDBOARD_SOUNDS_UPDATE',
```
Comment (vladfrangu) [inline deno/gateway/v10.ts:254]: Does this exist or not, I see no types that reference it

File: deno/payloads/v10/soundboard.ts [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+	 */
+	emoji_id: Snowflake | null;
+	/**
+	 * The unicode character of this sound's standard emoji
+	 */
+	emoji_name: string | null;
+	/**
+	 * The filename of this sound (for default sounds)
+	 */
+	override_path: string | null;
```
Comment (vladfrangu) [inline deno/payloads/v10/soundboard.ts:35]: The CDN route that lets you use this is not added in the PR

**Review by vladfrangu — CHANGES_REQUESTED** [review]

File: deno/gateway/v10.ts (hunk 1 above)
Comment (sdanialraza) [reply deno/gateway/v10.ts:254]: Right [here](https://github.com/discordjs/discord-api-types/pull/1113/files/ae2cd5c825dee99934b0ed5e5e605a8aa19aecb2#diff-4c8ded2885f17f9ae37adf936ac51bcf504c3b4e2394c016aae40d4329a96047R1358)? Or are you talking about some other types I missed

File: deno/payloads/v10/soundboard.ts (hunk 2 above)
Comment (sdanialraza) [reply deno/payloads/v10/soundboard.ts]: That property and CDN route are no longer documented, removed before the PR was merged

Comment (vladfrangu) [reply deno/payloads/v10/soundboard.ts]: https://discord.com/developers/docs/resources/soundboard#sound-files

Comment (sdanialraza) [reply deno/payloads/v10/soundboard.ts]: I added that one [here](https://github.com/discordjs/discord-api-types/pull/1113/files#diff-d94c4361c9ef564dcc1f291e8fa6742b21bfe7f75fb1642f39c7928fe034de4aR1357) but I don't see anything about `override_path` anymore

Comment (vladfrangu) [reply deno/payloads/v10/soundboard.ts]: Woops, I am as we say, le blind

**Review by vladfrangu — APPROVED** [review]

**Pull request merged**